pkg/appconsts/initial_consts.go

```diff
@@ -13,10 +13,6 @@ const (
 	// maximum number of bytes allowed in a valid block.
 	DefaultMaxBytes = DefaultGovMaxSquareSize * DefaultGovMaxSquareSize * ContinuationSparseShareContentSize
 
-	// DefaultGasPerBlobByte is the default gas cost deducted per byte of blob
-	// included in a PayForBlobs txn
-	DefaultGasPerBlobByte = 8
-
```

Comment:
nit: this is breaking so we should prob add ! to the PR title.

[no change needed] If we want to prep for the v3 release, we could also write a changelog entry on how consumers of this package need to modify their code if they used `DefaultGasPerBlobByte`. We could write these entires in a new markdown file in a new directory like `changelog/v3.md`

Reply:
https://github.com/celestiaorg/celestia-app/blob/main/docs/release-notes/pending-release.md now exists so we should add a line that states this constant was removed and what library consumers should use instead.